### ISSUE TITLE
Provide a way to clear / reset the log

### ISSUE THREAD

**Issue by Olical** [Olical/conjure#144]
A few people have asked, so it's probably about time :smile: I'm thinking a mapping like `<prefix>lR`,  so it requires a shift key to avoid accidental invocations. The pneumonic is `log Reset`, thoughts?

**Comment by tami5** [comment]
Agree. Also auto scroll to the bottom when evaluating something would be an awesome option to have

**Comment by Olical** [comment]
`<prefix>lR` will now delete the log buffer! If you have it open and you reset it you'll need to reopen it though, thoughts? Maybe I need two mappings, `lr` is a soft reset that deletes the lines but keeps the windows open and `lR` (the current one) which is a hard reset that completely rebuilds the buffer to fix any state issues.

**Comment by Olical** [comment]
Okay, there you go, `<prefix>lr` is a soft reset that leaves the windows intact and `<prefix>lR` is a hard reset that uses `bwipeout!` to completely destroy it. It's the nuclear option but might be good in some cases.

**Comment by Olical** [comment]
```viml
autocmd User ConjureEval if expand("%:t") =~ "^conjure-log-" | exec "normal G" | endif
```

Also, I'm working on #131 which adds custom events that Conjure dispatches when certain things happen. That includes when evals occur, you can use that with a buffer name check to execute a normal command (such as `G`) when you eval in the log buffer. @tami5 

This has the added benefit of letting you map these action autocmds to whatever behavior you want :smile: 

This is on `develop`, I'll smoke test it for a few days at work before publishing all of these commits. It'd be great if some people could try out these features and let me know if they work okay for you :blush: 

Enjoy!

**Comment by Olical** [comment]
Anyone else had a chance to try the log resetting or autocmds? I'd love to hear if it works on your setup.

It's been working for me, I even used the mapping in anger at work this week!

**Comment by leonampd** [comment]
@Olical I testing and for now, it's working well on my setup. Thanks! Conjure is awesome!

**Comment by Olical** [comment]
Released in [v4.10.0](https://github.com/Olical/conjure/releases/tag/v4.10.0)!
